include/unifex/let_value_with_stop_token.hpp

```diff
@@ -142,22 +142,25 @@ class _stop_token_sender<SuccessorFactory>::type {
       std::is_nothrow_constructible_v<SuccessorFactory, SuccessorFactory2&&>)
     : func_((SuccessorFactory2 &&) func) {}
 
-  template(typename Self, typename Receiver)
-    (requires same_as<remove_cvref_t<Self>, type> AND receiver<Receiver>) friend
-  auto tag_invoke(tag_t<unifex::connect>, Self&& self, Receiver&& r)
-    noexcept(
-      std::is_nothrow_invocable_v<member_t<Self, SuccessorFactory>, inplace_stop_token>
-      && std::is_nothrow_move_constructible_v<Receiver>
-      && std::is_nothrow_constructible_v<
-        operation<member_t<Self, SuccessorFactory>, unifex::remove_cvref_t<Receiver>>,
-        SuccessorFactory&&,
-        Receiver&&>)
-  -> operation<
-      SuccessorFactory,
-      unifex::remove_cvref_t<Receiver>> {
-    return operation<
-        SuccessorFactory,
-        unifex::remove_cvref_t<Receiver>>(
+  template(typename Self, typename Receiver)                                 //
+      (requires same_as<remove_cvref_t<Self>, type> AND receiver<Receiver>)  //
+      friend auto tag_invoke(
+          tag_t<unifex::connect>,
+          Self&& self,
+          Receiver&&
+              r) noexcept(std::
+                              is_nothrow_invocable_v<
+                                  member_t<Self, SuccessorFactory>,
+                                  inplace_stop_token>&& std::
+                                  is_nothrow_move_constructible_v<Receiver>&&
+                                      std::is_nothrow_constructible_v<
+                                          operation<
+                                              member_t<Self, SuccessorFactory>,
+                                              unifex::remove_cvref_t<Receiver>>,
+                                          SuccessorFactory&&,
+                                          Receiver&&>)
```

Comment:
Arguably uglier than manually formatted original :/

Reply:
Yeah. 😔  It's all about to change, though.